src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -141,7 +142,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String messageContent = request.getParameter("message");
 
     // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    // altered the whitelist so that text modifiers aren't removed
```

Comment:
You don't need to comment with what you changed. Maybe update the comment to explain what `Whitelist.basic()` matches though?